src/managers/message-user-queue-manager.js

```diff
@@ -0,0 +1,53 @@
+import { setKeyWithExpiryToLocalStore, getKeyFromLocalStore, setKeyToLocalStore } from '../utilities/local-storage';
+import { getHashedUserToken } from './user-manager';
+
+const messageQueueLocalStoreName = "gist.web.message.user";
+const MESSAGES_LOCAL_STORE_CACHE_IN_MINUTES = 60000 * 60;
```

Comment:
Just curious why we have different naming styles on constants (camelCase vs UPPER_CASE ?)

Reply:
Addressed this in the last commit.